test-case/manual/check-playback-capture-latency-at-the-beginning.md

```diff
@@ -0,0 +1,23 @@
+# Latency test
+Check latency at the beginning is manual test for piplines which isn't setup loopback for alsabat test
+
+# Preconditions
+SOF is loaded successfully
+
+# Test Description
+* Check playback latency at the beginning
+* Check capture latency at the begining
+
+## Test Case:
+1. `aplay -l` and `arecord -l` to check supported PCM
+2. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`, the pci device id can be find from `lspci` for Multimedia audio controller
+3. `aplay -Dhw:0,0 -r 48000 -f s16_le -c2 test.wav -vv -i` to trigger aplay from runtime PM suspended status, supported params can be dumped via `aplay -Dhw:0,0 --dump-hw-params`
+4. Check audiou output without obvious delay, latency at the begining is within 1s
```

Comment:
In this manual test, it is hard to tell 1s+ or not. Arecord is even harder.
typo audiou->audio.

Reply:
Still can observe the delay with “-vv”  param for process check